scheduler/src/test/scala/com/sky/kms/base/KafkaIntSpecBase.scala

```diff
@@ -4,26 +4,35 @@ import com.sky.kms.kafka.Topic
 import com.sky.kms.utils.RandomPort.randomPort
 import eu.timepit.refined.auto._
 import net.manub.embeddedkafka.Codecs.{nullDeserializer, stringDeserializer}
-import net.manub.embeddedkafka.ConsumerExtensions.{ConsumerRetryConfig, _}
-import net.manub.embeddedkafka.{Consumers, EmbeddedKafka, EmbeddedKafkaConfig}
+import net.manub.embeddedkafka.{EmbeddedKafka, EmbeddedKafkaConfig}
 import org.apache.kafka.clients.consumer.{ConsumerRecord, KafkaConsumer}
 import org.apache.kafka.common.serialization.Deserializer
 import org.scalatest.WordSpecLike
 
-trait KafkaIntSpecBase extends EmbeddedKafka with WordSpecLike with Consumers {
+import collection.JavaConverters._
+import scala.concurrent.duration._
+import scala.compat.java8.DurationConverters._
+
+trait KafkaIntSpecBase extends EmbeddedKafka with WordSpecLike {
 
   implicit lazy val kafkaConfig = EmbeddedKafkaConfig(kafkaPort = randomPort(), zooKeeperPort = randomPort())
 
   val scheduleTopic: Topic      = "scheduleTopic"
   val extraScheduleTopic: Topic = "extraScheduleTopic"
+  val pollTimeout               = 60 seconds
```

Comment:
```suggestion
  private val pollTimeout               = 60 seconds
```

Reply:
wouldn't you want that to be overridable (in general)

Reply:
I'd be happy with either `private val` or `def` (that can be overridden with a val or a def). If public, I'd like to see a name that expresses that it's a timeout for Kafka (e.g. `kafkaTimeout`, `kafkaOperationsTimeout`, ... [1]) otherwise in the overriding class you wouldn't be able to tell immediately what timeout you are configuring without finding the declaring class.

[1]: I find that using the word "poll" breaks encapsulation because externally clients of this interface shouldn't care about polling or how we consume those messages, but this is a detail. Even `subscribeAndPoll` should probably be made private if it's used only internally.

Reply:
* polITimeout => kafkaConsumerTimeout
   Originally called pollTimeout because the function was subscribeAndPoll, the kafka client's function is poll and Kafka is in the name of the base class. However, I agree it is too generic so I decided to use kafkaConsumerTimeout as there are other (consumer and/or producer) related timeouts as well that could be set up as well as the class just being called KafkaIntSpecBase. Let me know if that's ok.
* prefer the def over the private val or val.
* made subscribeAndPoll private